### ISSUE TITLE
nft-iptables support

### ISSUE THREAD

**Issue by sinancetinkaya** [brvphoenix/wrtbwmon#9]
OpenWRT has moved from iptables to nft-iptables.
Could you update wrtbwmon?

**Comment by lqh280** [comment]
Yes, I hope you can update iptables rules to nftables for openwrt 25.XX.
